docs/build/dependencies.md

```diff
@@ -0,0 +1,54 @@
+Dependency Management in ONNX Runtime
```

Comment:
Need format the title like add `#` to this line. Then update the level of remaining headers like add an extra `#` to most of them.